probe/awsecs/reporter.go

```diff
@@ -126,10 +149,11 @@ func (r Reporter) Tag(rpt report.Report) (report.Report, error) {
 		for serviceName, service := range ecsInfo.Services {
 			serviceID := report.MakeECSServiceNodeID(cluster, serviceName)
 			rpt.ECSService = rpt.ECSService.AddNode(report.MakeNodeWith(serviceID, map[string]string{
-				Cluster:             cluster,
-				ServiceDesiredCount: fmt.Sprintf("%d", service.DesiredCount),
-				ServiceRunningCount: fmt.Sprintf("%d", service.RunningCount),
-			}))
+				Cluster:               cluster,
+				ServiceDesiredCount:   fmt.Sprintf("%d", service.DesiredCount),
+				ServiceRunningCount:   fmt.Sprintf("%d", service.RunningCount),
+				report.ControlProbeID: r.probeID,
+			}).WithLatestActiveControls(ScaleUp, ScaleDown))
```

Comment:
Then with more reason. In the current state of things, reducing the number of tasks to 0 it would make the service disappear, right?

Then I would set a minimum number of tasks to 1 (not displaying the minus button if that's the case)